### PR TITLE
docs: Minor updates to installation pre-instructions

### PR DESCRIPTION
**Pull request by manikmagar**
Minor updates to the installation pre-instruction section.

### CODE REVIEW

File: docs/modules/ROOT/pages/installation.adoc [hunk 1]
```diff
@@ -18,9 +18,12 @@ To use `jbang` Java 8 is the minimum required version, however Java 11 or higher
 
 Note: `jbang` will download and install `java` from Adopt OpenJDK if no `java` is available.
 
-Once you hav installed from one of the below approaches it is recommended you run `jbang app setup`
- your `PATH` to include jbang app scripts + it will on operating systems that supports
-it setup a `j!` alias you can use instead of `jbang`.
+There are multiple ways to install jbang. See below for all available options.
+
+Once you have jbang installed, it is recommended you run `jbang app setup`. This script will make following changes on your sysmte -
```
Comment (maxandersen) [inline docs/modules/ROOT/pages/installation.adoc:23]:
```suggestion
Once you have jbang installed, it is recommended you run `jbang app setup`. This script will make following changes on your system:
```

**Review by maxandersen — CHANGES_REQUESTED** [review]
Looks good just a small typo to fix before merge

File: docs/modules/ROOT/pages/installation.adoc (hunk 1 above)
Comment (manikmagar) [reply docs/modules/ROOT/pages/installation.adoc]: Now corrected in -  https://github.com/jbangdev/jbang/pull/1286/commits/10c49361cc6f8a3c51375800df7ffe6e42ded601.

**Comment by maxandersen** [comment]
@all-contributors please add @manikmagar  for code

**Pull request merged**